### PR TITLE
correct indel pos

### PR DESCRIPTION
**Pull request by qinqian**

### CODE REVIEW

File: depmapomics/mutations.py [hunk 1]
```diff
@@ -460,3 +461,199 @@ def generateGermlineMatrix(
...
+    #         remove variants without gene symbols
+    #         sort variants by genome position to be compatible with IGV
+    maf = standardize_maf(maf)
+    maf = maf.loc[~maf.Variant_Classification.isin(['Silent', 'RNA', 'Intron', "5'UTR", "3'Flank", 'Splice_Region', "5'Flank"]), :]
+    maf = maf.loc[~maf.Hugo_Symbol.isnull(), :]
+    maf = maf.sort_values(by=["Chromosome", "Start_Position", "End_Position"])
+
+    # optional step: add metadata information
+    # step 4: remove high af from DepMap cohort
+    internal_afs_23q4 = maf.loc[:, ['Chromosome', 'Start_Position', 'End_Position', 'Tumor_Seq_Allele1', 'Tumor_Seq_Allele2']].apply(lambda x: ':'.join(map(str, x)), axis=1)
```
[the remark this answers is not in the document]
Comment (qinqian) [reply depmapomics/mutations.py:649]: add a input for release number

File: depmapomics/mutations.py [hunk 2]
```diff
@@ -460,3 +461,199 @@ def generateGermlineMatrix(
...
+    # step 1: filter the leftmost synonymous mutation 
+    maf = maf.query("~variant_info.str.contains('^synony', regex=True)")
+    
+    # step 2: less stringent cutoff for gnomad
+    maf = maf.drop(maf.index[((maf.gnomadg_af > adjusted_gnomad_af_cutoff) | (maf.gnomade_af > adjusted_gnomad_af_cutoff))], axis=0)
+
+    # step 3: format the mutation and remove all silent mutation classes
+    #         remove variants without gene symbols
+    #         sort variants by genome position to be compatible with IGV
+    maf = standardize_maf(maf)
```
[the remark this answers is not in the document]
Comment (qinqian) [reply depmapomics/mutations.py:642]: I added the code into dm_omics.py to replace the old converters

**Comment by qinqian** [comment]
This may need some debugging to see if it is compatible with the old codes.

File: depmapomics/config_dev.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+version = '23Q4'
```
Comment (5im1z) [inline depmapomics/config_dev.py:1]: This is already in constants.py, so no need to add them in here and config_prod again. We only need to make the variable in constants.py a default input for the function you added. I'll approve and fix that myself. Thanks!

**Review by 5im1z — APPROVED** [review]

**Pull request merged**